### PR TITLE
Add a property to control persistence of topic messages

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:
* Add a property to control persistence of topic messages
* Change Rosetta single image to disable persistence of topic messages by default

**Related issue(s)**:

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Nana-EC — DISMISSED** [review]
LGTM

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListenerTopicTest.java [hunk 1]
```diff
@@ -453,6 +453,39 @@ void submitMessageTestTopicNotFound() {
...
+        var transaction = createSubmitMessageTransaction(topicId, message, chunkNum, chunkTotal, initialTransactionId);
+        var transactionRecord = createTransactionRecord(topicId, sequenceNumber, runningHash
+                .getBytes(), runningHashVersion, consensusTimestamp, responseCode);
+
+        // when
+        parseRecordItemAndCommit(new RecordItem(transaction, transactionRecord));
+
+        // then
+        assertEquals(0L, entityRepository.count());
+        assertEquals(0L, topicMessageRepository.count());
```
Comment (xin-hedera) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListenerTopicTest.java:485]: add assert to ensure the transaction is saved to the `transaction` table?

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListenerTopicTest.java (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListenerTopicTest.java:485]: Added

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**